src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -296,7 +391,7 @@ class XrdAccSciTokens : public XrdAccAuthorize, public XrdSciTokensHelper
         pthread_rwlock_init(&m_config_lock, nullptr);
```

Comment:
You should consider using XrdSysRWLock as this would be very helpful in debugging lock/unlock problems in the future.